tests/testthat/test-db_query_fields.R

```diff
@@ -71,7 +71,7 @@ test_that('db_query_fields works with mock', {
         request_body=paste0(
           '^SELECT \\* FROM \\(',
             '\\(SELECT 1 AS a, \'t\' AS b\\) "a"',
-          '\\) "zzz[0-9]+" LIMIT 0$'
+          '\\) "zzz[0-9]+" WHERE 1 = 0$'
```

Comment:
why is this better? it could also be `WHERE false`

Reply:
It's not necessarily better. I just mirrored the dbplyr [approach](https://github.com/tidyverse/dbplyr/blob/c0e51957a2fbeb4fe7bc93f0ed677d6d33e14828/R/dbi-s3.r#L132).